storagetransfer/conftest.py

```diff
@@ -46,61 +64,88 @@ def parse_and_cache_secret_json(payload: str):
     - ``{"AccessKeyId": "", "SecretAccessKey": ""}``
     """
 
-    global aws_secret_manager_cache
+    global aws_secret_cache
```

Comment:
Is there any reason this can't be a fixture instead of a global variable?

Reply:
Refactored this to a module-scoped fixture. Also fixed the existing fixtures (most should be function-scoped).